<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDialog</class>
 <widget class="QDialog" name="PatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>391</width>
     <height>93</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lastName">
      <property name="text">
       <string>Прізвище</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="firstName">
      <property name="text">
       <string>Ім'я</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="diag">
      <property name="text">
       <string>Діагноз</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lastNameEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="firstNameEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="diseaseEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="createButton">
      <property name="text">
       <string>Створити</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
